spec/factories/requests.rb

```diff
@@ -3,7 +3,10 @@
 FactoryBot.define do
   factory :request do
     requester { create :person }
-    point_person { create :techie }
     description { Faker::TvShows::Buffy.quote }
+
+    trait :with_techie do
+      point_person { create :techie }
```

Comment:
We should avoid explicit `create` and `build` if we can. I think this will work:

```suggestion
      point_person factory: :techie
```